<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-01T22:15:19 IST" name="Default suite" finished-at="2024-01-01T22:15:43 IST" duration-ms="23051">
    <groups>
      <group name="regression">
        <method signature="ClientTest.verifyclient()[pri:1, instance:Test.ClientTest@21e360a]" name="verifyclient" class="Test.ClientTest"/>
      </group> <!-- regression -->
    </groups>
    <test started-at="2024-01-01T22:15:19 IST" name="Default test" finished-at="2024-01-01T22:15:43 IST" duration-ms="23051">
      <class name="Test.ClientTest">
        <test-method is-config="true" signature="beforeMethod(java.lang.String)[pri:0, instance:Test.ClientTest@21e360a]" started-at="2024-01-01T22:15:20 IST" name="beforeMethod" finished-at="2024-01-01T22:15:36 IST" duration-ms="16191" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyclient()[pri:1, instance:Test.ClientTest@21e360a]" started-at="2024-01-01T22:15:36 IST" name="verifyclient" finished-at="2024-01-01T22:15:40 IST" duration-ms="4380" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Utility.ExcelRead.getCellValue(ExcelRead.java:47)
at Utility.ExcelRead.getDataFromExcel(ExcelRead.java:65)
at Test.ClientTest.verifyclient(ClientTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyclient -->
        <test-method is-config="true" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:Test.ClientTest@21e360a]" started-at="2024-01-01T22:15:40 IST" name="takeScreenShotOnFailure" finished-at="2024-01-01T22:15:43 IST" duration-ms="2318" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyclient status=FAILURE method=ClientTest.verifyclient()[pri:1, instance:Test.ClientTest@21e360a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
      </class> <!-- Test.ClientTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
